pkg/session/session.go

```diff
@@ -3836,7 +3846,7 @@ func getStoreBootstrapVersion(store kv.Storage) int64 {
 	ctx := kv.WithInternalSourceType(context.Background(), kv.InternalTxnBootstrap)
 	err := kv.RunInNewTxn(ctx, store, false, func(_ context.Context, txn kv.Transaction) error {
 		var err error
-		t := meta.NewMutator(txn)
+		t := meta.NewReader(txn)
```

Comment:
Does this line of change related this pr？

Reply:
no, just found and change it by the way

Reply:
Got it！